{"uuid":"6fa780cc-4ea9-4886-aa90-4d85b0e58d10","historyId":"8bafcb88579f444d05db128ef3435359","fullName":"org.example.Ex14_TestNG.Assertions.SoftAssertions.testSoftAssertions","labels":[{"name":"package","value":"org.example.Ex14_TestNG.Assertions.SoftAssertions"},{"name":"testClass","value":"org.example.Ex14_TestNG.Assertions.SoftAssertions"},{"name":"testMethod","value":"testSoftAssertions"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Selenium"},{"name":"subSuite","value":"org.example.Ex14_TestNG.Assertions.SoftAssertions"},{"name":"host","value":"VishalNivangune"},{"name":"thread","value":"10260@VishalNivangune.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"testSoftAssertions","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\t❌ Count mismatch! expected [10] but found [5],\n\t❌ Error should not be visible! expected [false] but found [true],\n\t❌ Username should be null! expected [null] but found [Admin]","trace":"java.lang.AssertionError: The following asserts failed:\n\t❌ Count mismatch! expected [10] but found [5],\n\t❌ Error should not be visible! expected [false] but found [true],\n\t❌ Username should be null! expected [null] but found [Admin]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\r\n\tat org.example.Ex14_TestNG.Assertions.SoftAssertions.testSoftAssertions(SoftAssertions.java:58)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1751340728890,"stop":1751340728921}